06-reactjs/components/movie-store.js

```diff
@@ -0,0 +1,76 @@
+var React = require('react');
```

Comment:
Not sure if *"store"* is a happy name when using Flux to design a component and not an actual Store, right @troianoandres? :P